src/clients/firewall_stage1_proposal.rb

```diff
@@ -76,7 +76,9 @@ def main
         # Don't override users settings
         SuSEFirewall4Network.prepare_proposal unless SuSEFirewallProposal.GetChangedByUser
 
-        adjust_configuration
+        # this method is not easily mockable in rspec and currently is out of scope
+        # for testing in firewall_stage1_proposal_test.rb
+        adjust_configuration if !Mode.test
```

Comment:
That is done by a specific ` allow(Mode).to receive(:test).and_return(true)` for that test ;-)
I guess the mocking is problematic because this is a client.